src/Time/StepChoosers/ErrorControl.hpp

```diff
@@ -293,54 +293,35 @@ PUP::able::PUP_ID
 }  // namespace StepChoosers
 
 namespace Tags {
-namespace detail {
-template <typename Tag>
-using NoPrefix = Tag;
-}  // detail
-
 /// \ingroup TimeGroup
-/// \brief Base tag for reporting whether the `ErrorControl` step chooser is in
+/// \brief Tag for reporting whether the `ErrorControl` step chooser is in
 /// use.
-struct IsUsingTimeSteppingErrorControlBase : db::BaseTag {};
+struct IsUsingTimeSteppingErrorControlBase : db::SimpleTag {
```

Comment:
Should this get the `Base` at the end of the name removed and then we add `Compute` to the end of the compute tag? That would match our general convention for these

Reply:
Good idea.  Done and squashed.